### PR TITLE
fix: Zentral logo dark/light mode

### PR DESCRIPTION
**Pull request by mazzi**
Using CSS.

### CODE REVIEW

File: server/static_src/scss/styles.scss [hunk 1]
```diff
@@ -182,12 +182,16 @@ pre.scrolleable {
     display: block;
 }
 
-.zentral-logo.light {
-    background-image: url(../images/logo-light.svg);
+[data-bs-theme=light] {
+    .zentral-logo {
```
Comment (np5) [inline server/static_src/scss/styles.scss:186]: Please do not use nested CSS, especially for one nested rule.

File: server/static_src/scss/styles.scss [hunk 2]
```diff
@@ -182,12 +182,16 @@ pre.scrolleable {
...
+        background-image: url(../images/logo-light.svg);
+    }
 }
 
-.zentral-logo.dark {
-    background-image: url(../images/logo-dark.svg);
+[data-bs-theme=dark] {
+    .zentral-logo {
```
Comment (np5) [inline server/static_src/scss/styles.scss:192]: Same

**Review by np5 — CHANGES_REQUESTED** [review]

File: server/static_src/scss/styles.scss (hunk 1 above)
Comment (mazzi) [reply server/static_src/scss/styles.scss:186]: This will grow, and this is `scss`, but as you wish.

Comment (np5) [reply server/static_src/scss/styles.scss]: Yes, but nesting is still a young feature and we do not really need it since we already have a SCSS pipeline.

**Review by np5 — APPROVED** [review]

**Pull request merged**